README.md

```diff
@@ -8,7 +8,7 @@ tools.
 ## Contributing to Mozilla Loads
 
 Interested in contributing to the development of Mozilla's Loads project?
-Documentation on Loads and how to get involved can be found at the [Loads Project documentation](https://loads.readthedocs.org/en/0.2).
+Documentation on Loads and how to get involved can be found at the [Loads Project documentation](http://loadsv2.rtfd.org/).
```

Comment:
Sadly, we can't use an HTTPS URL here. We could switch to the unshortened URL which supports HTTPS: https://loadsv2.readthedocs.org/en/latest/

(Or, we could try filing a bug with whomever manages rftd.org and see if they can support HTTPS, which may be a bit silly for a redirect URL site)

---

**UPDATE:** Filed https://github.com/rtfd/readthedocs.org/issues/1952 upstream to see if an HTTPS short URL is possible.